tests/users/test_oauth_login.py

```diff
@@ -54,6 +55,54 @@ def test_unsupported_provider(self):
         self.assertEqual(res.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(res.data['error'], "Unsupported OAuth provider")
 
+    @patch('users.views.oauth_view.load_backend')
+    def test_google_oauth_sets_user_active_based_on_email_verified(self, mock_load_backend):
+        """Test Google OAuth login sets is_active correctly based on email_verified"""
+        mock_backend = MagicMock()
+
+        active_user = self.oauth_user
+        active_user.first_name = "Updated"
+        active_user.last_name = "Name"
+        mock_backend.do_auth.return_value = active_user
+
+        mock_load_backend.return_value = mock_backend
+
+        res = self.client.post(self.auth_url, {'provider': 'google', 'access_token': 'token'}, format='json')
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        user = User.objects.get(email='oauth@example.com')
+        self.assertEqual(user.first_name, 'Updated')
+        self.assertEqual(user.last_name, 'Name')
+        self.assertTrue(user.is_active)
+        
+        access_token = res.data['access']
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION=f'Bearer {access_token}')
+        protected_response = client.get('/api/v1/auth/me/')
+        self.assertEqual(protected_response.status_code, 200)
+
+    @patch('users.views.oauth_view.load_backend')
+    def test_github_oauth_sets_user_inactive(self, mock_load_backend):
+        """Test GitHub OAuth login sets is_active correctly based on email_verified and ensures that access token is not issued"""
+        inactive_user = User.objects.create(
+            email='test@example.com',
+            first_name='Test',
+            last_name='User',
+            role = self.role,
+            is_active=False
+        )
+
+        mock_backend = MagicMock()
+        mock_backend.do_auth.return_value = inactive_user
+        mock_load_backend.return_value = mock_backend
```

Comment:
Tests mock the backend's do_auth return value but do not verify that the backend was called with expected arguments. Add an assertion for call args.
```
mock_backend.do_auth.assert_called_with('token')
```